homeassistant/components/switch/__init__.py

```diff
@@ -31,6 +32,7 @@
     discovery.services.BELKIN_WEMO: 'wemo',
     wink.DISCOVER_SWITCHES: 'wink',
     isy994.DISCOVER_SWITCHES: 'isy994',
+    camera.DISCOVER_SWITCHES: 'child_switch',
```

Comment:
Duh, not sure why I didn't see this, I'm adding the constant to switch rather than the const file, does that make the most sense? or would you do it the other way around?